lib/locales/ta.ts

```diff
@@ -0,0 +1,998 @@
+    charles_of_jesus_de_foucauld: 'முத்திப்பேறுபெற்ற சார்ல்ஸ் டி புகொல்ட் - அருட்பணியாளர்',
+    charles_spinola_and_jerome_de_angelis_priests:
+      'முத்திப்பேறுபெற்றவர்கள் சார்ல்ஸ் ஸ்பினோலா, ஜெரோம் டி ஏஞ்சலிஸ் - அருட்பணியாளர்கள், மறைசாட்சிகள்',
+    charles_spinola_priest: 'முத்திப்பேறுபெற்ற சார்ல்ஸ் ஸ்பினோலா - அருட்பணியாளர், மறைசாட்சி',
+    christopher_magallanes_priest_and_companions_martyrs:
+      'புனித கிறிஸ்டோபர் மகல்லானஸ் - அருள்பணியாளர்; தோழர்கள் - மறைச்சாட்சியர்',
+    christopher_of_palestine_martyr: 'புனித கிறிஸ்டோபர் - மறைச்சாட்சி',
+    ciaran_of_clonmacnoise_abbot: 'புனித சைரன் - மடாதிபதி',
+    clare_of_assisi_virgin: 'புனித கிளாரம்மாள் - கன்னி',
+    clement_i_pope: 'புனித முதலாம் கிளமெண்ட் - திருத்தந்தை, மறைசாட்சி',
+    clement_mary_hofbauer_priest: 'புனித கிளெமெண்ட் மரிய ஹொப்பர் - அருட்பணியாளர்',
+    clement_of_ohrid_and_gorazd_of_moravia_bishops_and_companions:
+      'ஒறிட்டின் புனித க்ளெமெண்ட், புனித கொராஸ்ட் - ஆயர்கள், தோழர்கள',
+    clotilde_of_burgundy: 'புனித க்ளொடில்டா',
+    colman_of_cloyne_bishop: 'க்லோனின். புனித கோல்மன் - ஆயர்',
+    colman_of_dromore_bishop: 'ட்றொமோரின் புனித கோல்மன் - ஆயர்',
+    colman_of_kilmacduagh_bishop: 'கில்மக்டக்கின் புனித கோல்மன் - ஆயர்',
+    columba_marmion_priest: 'முத்திப்பேறுபெற்ற கொலம்பா மார்மியன் - அருட்பணியாளர்',
+    columba_of_iona_abbot: 'புனித கொலம்பா - மடாதிபதி',
+    columba_of_iona_abbot_copatron_of_ireland: 'புனித கொலம்பா - மடாதிபதி, மறை அறிவிப்பாளர், அயர்லாந்தின் பாதுகாவலர்',
+    columban_of_luxeuil_abbot: 'புனித கொலம்பன் - மடாதிபதி',
+    comgall_of_bangor_abbot: 'புனித கொம்கெல் - மடாதிபதி',
+    commemoration_of_all_the_faithful_departed: 'இறந்த ஆத்துமாக்களின் நினைவு நாள்',
+    conleth_of_kildare_bishop: 'புனித கொன்லெத் - ஆயர்',
+    conrad_of_constance_and_gebhard_of_constance_bishops: 'புனித கொன்ராட், கொன்ஸ்டன்ஸின் கெபார்ட் - ஆயர்கள்',
+    conrad_of_parzham_religious: 'பார்சத்தின் புனித கொன்ராட் - துறவி',
+    conversion_of_saint_paul_the_apostle: 'திருத்தூதர் புனித பவுலின் மனமாற்றம் - விழா',
+    corbinian_of_freising_bishop: 'புனித கொர்பினியன் - ஆயர்',
+    cornelius_i_pope_and_cyprian_of_carthage_bishop_martyrs:
+      'புனித கொர்னேலியுஸ் - திருத்தந்தை, புனித சிப்ரியன் - ஆயர், மறைசாட்சிகள்',
+    cosmas_of_cilicia_and_damian_of_cilicia_martyrs: 'புனிதர்கள் கொஸ்மஸ், தமியான் - மறைசாட்சிகள்',
+    cuthbert_of_lindisfarne_bishop: 'புனித கத்பேர்ட் - ஆயர்',
+    cyril_constantine_the_philosopher_monk_and_methodius_michael_of_thessaloniki_bishop:
+      'புனித சிரில் - துறவி, புனித் மெதோடியஸ் - ஆயர்',
+    cyril_constantine_the_philosopher_monk_and_methodius_michael_of_thessaloniki_bishop_copatrons_of_europe:
+      'புனித சிரில் - துறவி, புனித மெதோடியஸ் - ஆயர், ஐரோப்பாவின் பாதுகாவலர்கள்',
+    cyril_constantine_the_philosopher_monk_and_methodius_michael_of_thessaloniki_bishop_slavic_missionaries_copatrons_of_europe:
+      'புனித சிரில், புனித் மெதோடியஸ் - சிலாவிக்கின் மறை போதகர்கள்',
```

Comment:
Could you use use `Slavic` word (`ஸ்லாவ்`) from [this article](https://www.vaticannews.va/ta/pope/news/2023-10/pope-audience-slav-saints-cyril-methodius.html) please? I believe in the key value, you used a different word (`சிலாவிக்கின்`) Or do I get wrong? 🤔